compiler/il/VectorOperations.enum

```diff
@@ -882,7 +882,10 @@ VECTOR_OPERATION_MACRO(\
    /* .reverseBranchOperation  = */ TR::vBadOperation, \
    /* .booleanCompareOpCode    = */ TR::BadILOp, \
    /* .ifCompareOpCode         = */ TR::BadILOp, \
-   /* .description             =    vector blend witn mask operator */ \
+   /* .description             =    vector blend operator */ \
+   /*                               blends second vector into first based on mask (third child) */ \
+   /*                               if lane is not set in the mask the value from the first vector is chosen */ \
+   /*                               the lane from the second vector is chosen otherwise */
```

Comment:
I'm sorry to nit about something like this, but the commenting style is a bit awkward.  I'm not clear why you use `\` at the end of each comment line except the last one, and why each line has its own comment.

Something like you have for the `vcompressbits` opcode seems more visually appealing (to me at least).

Reply:
I took it from here: https://github.com/eclipse-omr/omr/blob/0dcc2399abd9678720060e70ac8982f16c645818/compiler/il/OMROpcodes.enum#L7204

Reply:
But you are right `vcompressbits` looks better. Let me change.